src/hooks/useWalletsWithBalancesAndNames.js

```diff
@@ -10,7 +10,7 @@ export default function useWalletsWithBalancesAndNames() {
   const walletsWithBalancesAndNames = useMemo(
     () =>
       mapValues(wallets, wallet => {
-        const updatedAccounts = map(wallet.addresses, account => ({
+        const updatedAccounts = wallet.addresses.map(account => ({
```

Comment:
worth to check if the actually have addresses here 
```suggestion
        const updatedAccounts = (wallet.addresses ?? []).map(account => ({
```

Reply:
if we received a `wallet`, then `addresses` is by default an empty array 
But yea, to keep it safety better to make this check